wscript

```diff
@@ -969,6 +969,7 @@ def configure(ctx):
         ctx.options.enable_lua = True
 
     if ctx.options.SWIFT_FLAGS:
+        print(split(ctx.options.SWIFT_FLAGS))
```

Comment:
leftover debug print?

Reply:
yeah, was still testing yesterday and the build didn't finish before i went to bed.